### DISCUSSION TITLE
Does Auro works with React-Native?, iONIC/Capacitor? Kotlin/Swift?

### DISCUSSION THREAD

**Discussion by aa-ashukla** [AlaskaAirlines/WC-Generator#547] [General Auro discussions]
We are looking for a new framework for our app. On my way of researching new frameworks, I came to know about Auro from my Dev Manager. On the Auro page I don't see any discussion on mobile apps, running natively on devices.

**Comment by blackfalcon** [comment]
Auro Web Components have been designed and engineered to cater to the web development landscape. Their primary purpose is to ensure interoperability across various web-based platforms, including React, Vue, Svelte, and Angular. These components provide a consistent set of UI elements, promoting a unified user experience.

However, it's essential to recognize that Auro Web Components are not inherently compatible with native iOS or Android environments. Their foundation relies on HTML5 Web Components, which operate within web browsers. As a result, direct integration with React Native—a framework specifically tailored for native mobile app development—poses challenges due to fundamental differences in rendering approaches.

**SSR/ESM Compliance and the Auro Ecosystem:**

On a positive note, a significant portion of Auro Web Components adheres to best practices, including Server-Side Rendering (SSR) and ECMAScript Modules (ESM) compliance. This ensures optimal performance and seamless integration within modern web applications. You can find further details in this [GitHub Discussion](https://github.com/orgs/AlaskaAirlines/discussions/502).

**Exploring Ionic and Capacitor:**

When it comes to Ionic and Capacitor, there's potential for synergy. Both frameworks operate at the intersection of web and native development. Capacitor, in particular, acts as a bridge by wrapping web apps in a WebView, allowing access to native device features. Considering this, incorporating HTML5 Web Components into an Ionic or Capacitor project becomes feasible.

**Reply by aa-ashukla** [reply]
thanks. i was hoping to that there would be synergy between auro and Ionic/Cap projects. This is very helpful.
